project/src/main/ui/chat/chat-library.gd

```diff
@@ -10,39 +10,25 @@ into the UI.
 # This version number follows a 'ymdh' hex date format which is documented in issue #234.
 const DIALOG_DATA_VERSION := "15d2"
 
-"""
-Loads the chat events for the currently focused chatter.
-
-Returns an array of ChatEvent objects for the dialog sequence which the player should see.
-"""
-func load_chat_events() -> ChatTree:
-	var chat_tree
-	var focused: Node = ChattableManager.get_focused()
-	if not focused.has_meta("chat_id"):
-		# can't look up chat events without a chat_id; return an empty array
-		push_warning("Chattable %s does not define a 'chat_id' property." % focused)
-	else:
-		# open the json file for the currently focused chatter
-		var chat_id: String = focused.get_meta("chat_id")
-		chat_tree = load_chat_events_from_file("res://assets/main/dialog/%s.json" % chat_id)
-	return chat_tree
```

Comment:
This moved method is worth describing in a commit comment.

Reply:
Commit comment now mentions `Moved load_chat_events function from ChatLibrary into ChattableManager`